.babelrc

```diff
@@ -1,18 +1,3 @@
 {
-  "presets": [
-    [
-      "env",
-      {
-        "modules": false
-      }
-    ]
-  ],
-  "env": {
-    "development": {
-      "plugins": ["external-helpers"]
-    },
-    "test": {
-      "plugins": ["transform-es2015-modules-commonjs"]
-    }
-  }
+  "presets": ["@babel/preset-env", "@babel/preset-typescript"]
```

Comment:
You might also need `@babel/plugin-proposal-class-properties` and `@babel/plugin-proposal-object-rest-spread` if you use these features.

Reply:
Yeah, I'd prefer not to add them until we need them. We will get a nice error message telling us which plugin to add, should the need arise in the future.